eng/Versions.props

```diff
@@ -98,14 +98,13 @@
     <MicrosoftBuildFrameworkPackageVersion>16.8.0</MicrosoftBuildFrameworkPackageVersion>
     <MicrosoftBuildLocatorPackageVersion>1.2.6</MicrosoftBuildLocatorPackageVersion>
     <MicrosoftBuildPackageVersion>16.8.0</MicrosoftBuildPackageVersion>
-    <MicrosoftInternalVisualStudioShellEmbeddablePackageVersion>17.0.0-preview-1-30928-1111</MicrosoftInternalVisualStudioShellEmbeddablePackageVersion>
     <MicrosoftNETCoreApp50PackageVersion>$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)</MicrosoftNETCoreApp50PackageVersion>
     <!-- Packages from dotnet/roslyn -->
     <MicrosoftCodeAnalysisAnalyzerTestingPackageVersion>$(Tooling_MicrosoftCodeAnalysisTestingVersion)</MicrosoftCodeAnalysisAnalyzerTestingPackageVersion>
     <MicrosoftCodeAnalysisTestingVerifiersXunitPackageVersion>$(Tooling_MicrosoftCodeAnalysisTestingVersion)</MicrosoftCodeAnalysisTestingVerifiersXunitPackageVersion>
     <MicrosoftNetCompilersToolsetPackageVersion>3.9.0-2.20573.10</MicrosoftNetCompilersToolsetPackageVersion>
     <MicrosoftServiceHubFrameworkPackageVersion>2.7.2466</MicrosoftServiceHubFrameworkPackageVersion>
-    <MicrosoftVisualStudioComponentModelHostPackageVersion>$(MicrosoftVisualStudioPackagesVersion)</MicrosoftVisualStudioComponentModelHostPackageVersion>
+    <MicrosoftVisualStudioComponentModelHostPackageVersion>17.0.30-g62d2639511</MicrosoftVisualStudioComponentModelHostPackageVersion>
```

Comment:
This is incorrect. We don't have a correct version for you yet. Watch for an update on the table and/or the Teams channel for the version to use.

Reply:
Figured there was something missing there. Things restored and compiled with this one though 😄

Reply:
@NTaylorMullen there's a "correct" version on the OneNote now.